### ISSUE TITLE
hugging face giving error

### ISSUE THREAD

**Issue by BernardoOlisan** [EleutherAI/vqgan-clip#14]
hello, I want to use the hugging face api https://huggingface.co/spaces/EleutherAI/VQGAN_CLIP of vqgan+clip, but is giving me an error everytime I use it
![image](https://user-images.githubusercontent.com/46250121/167267798-824ea8c4-74b0-44b8-86ba-167fa63f8fac.png)

**Comment by StellaAthena** [comment]
Thank you for the report. The issue seems to be something on HuggingFace's end... I'll look into it.

**Comment by BernardoOlisan** [comment]
@StellaAthena okey, thank you for the answer, can I get notify when the error is gone? I need to use this api for a project...

**Comment by StellaAthena** [comment]
@BernardoOlisan There is a bot in the EleutherAI discord server where you can use this model. Go to `#the-faraday-cage` and type `.imagine sketch of a 3D printer by Leonardo da Vinci` to get an image generated. Generation takes about three minutes and runs on our GPUs for free.

**Comment by BernardoOlisan** [comment]
> 

yes but, the thing is that I want to use the api from hugging face:/

**Comment by StellaAthena** [comment]
@BernardoOlisan is there a particular reason you wish to use that API?

You can also copy the code underlying the demo and run it on a local GPU or in a Google Colab notebook.

**Comment by BernardoOlisan** [comment]
> @BernardoOlisan is there a particular reason you wish to use that API?
> 
> 
> 
> You can also copy the code underlying the demo and run it on a local GPU or in a Google Colab notebook.

Because a company wants to use this technology using JavaScript, so huggingface provides it, is a website with NEXT.js I need to use it with JS

**Comment by BernardoOlisan** [comment]
@StellaAthena any idea?

**Comment by BernardoOlisan** [comment]
@StellaAthena 
![image](https://user-images.githubusercontent.com/46250121/167426893-efce8c17-d4d3-4430-8b9f-f3f81f974f92.png)
so I clone the repo in hugging face, and this is the error, any idea?

**Comment by StellaAthena** [comment]
No I have no idea what that means

**Comment by BernardoOlisan** [comment]
@StellaAthena I solved it, I also created a new huggingface space https://huggingface.co/spaces/BernardoOlisan/vqganclip and made a pull request here https://github.com/EleutherAI/vqgan-clip/pull/15
:)
